src/java/org/apache/cassandra/utils/memory/BufferPool.java

```diff
@@ -252,9 +268,7 @@ private Chunk allocateMoreChunks()
                 noSpamLogger.error("Buffer pool failed to allocate chunk of {}, current size {} ({}). " +
                                    "Attempting to continue; buffers will be allocated in on-heap memory which can degrade performance. " +
                                    "Make sure direct memory size (-XX:MaxDirectMemorySize) is large enough to accommodate off-heap memtables and caches.",
-                                   prettyPrintMemory(MACRO_CHUNK_SIZE),
-                                   prettyPrintMemory(sizeInBytes()),
-                                   oom.toString());
+                                   READABLE_MACRO_CHUNK_SIZE, readableMemoryUsage, oom.getClass().getName());
```

Comment:
It didn't look like the `OutOfMemoryError` had much of a message accompanying it...really ever. If it did, it basically looked like the one already explicitly logged above. Without a message the class name is all we got before, so now we're just doing it cheaply.